src/algebra/regular.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2021 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import algebra.group
+
+/-!
+# Regular elements
+
+We introduce left-regular, right-regular and regular elements.  The final goal is to develop
+part of the API to prove, eventually, results about non-zero-divisors.
```

Comment:
Maybe mention explicitly in this module docstring that a regular element in a commutative ring is (by definition) a non-zero-divisor.

Reply:
Done!